.gitignore

```diff
@@ -12,3 +12,11 @@ release*/
 .DS_Store
 .version
 \.scannerwork/
+
+/snap/.snapcraft/
+/parts/
+/stage/
+/prime/
+/*.snap
+/*_source.tar.bz2
```

Comment:
I don't believe this is a snap artifact

Reply:
This file is created while running `snapcraft cleanbuild`, snapcraft will pack a copy of the source tree archive before pushing it to the remote LXD container.